### DISCUSSION TITLE
bug: The app is not working on iOS simulator, shows could not find a storyboard named 'Main'

### DISCUSSION THREAD

**Discussion by aakatheeri** [ionic-team/capacitor#3876] [Q&A]
### Capacitor Version

```
Latest Dependencies:

  @capacitor/cli: 2.4.3
  @capacitor/core: 2.4.3
  @capacitor/android: 2.4.3
  @capacitor/electron: 2.4.3
  @capacitor/ios: 2.4.3

Installed Dependencies:

  @capacitor/android not installed
  @capacitor/cli 2.4.3
  @capacitor/core 2.4.3
  @capacitor/ios 2.4.3
  @capacitor/electron not installed
```

### Platform(s)

- Apple macOS version: **10.15.7**
- xCode version: **11.2.1**
- iOS version (Simulator): **14.0.1**

### Current Behavior

1. I created a blank ionic project
2. Built the project
3. Added iOS version through this command
```
ionic cap add ios
```
4. Opened the iOS project
```
ionic cap open ios
```
5. I run the app to work on iOS simulator 'iPhone 11 Pro, iOS version ~ 14.0.1'.
6. The app shows black screen, then crashed and shows this error on xCode console:
```
Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: 'Could not find a storyboard named 'Main' in bundle NSBundle
``` 

### Expected Behavior

The app should run on iOS simulator without any issue.

### Other Technical Details

`npm --version` output:

`node --version` output:

`pod --version` output (iOS issues only):

### Additional Context

I tried to solve this problem by removing the 'Main' storyboard file on Info.list as mentioned on below link, but it's not work even!
https://stackoverflow.com/questions/19240054/could-not-find-a-storyboard-named-main-in-bundle

**Comment by jcesarmobile** [comment]
I can't reproduce.
By the answers on that link, looks like it can happen if the storyboard gets corrupted somehow, try removing ios folder and running `ionic cap add ios` again.
Do not remove the Main storyboard, that answer is for iOS 13+ apps that use UIScenes, Capacitor doesn't use UIScenes because they don't work on iOS 11 nor iOS 12 and Capacitor still support them.

Moving to discussions so maybe somebody else can help you.

**Comment by aakatheeri** [comment]
@jcesarmobile The problem has been fixed!

I tried to remove the 'ios' folder but didn't work for me. What I did is remove @ionic/cli npm package and cocoapods from the macOS. Then I install them again and restart the device. I created a fresh new Ionic app and then build a project to add the iOS platform 'ionic cap add ios', it works!

I have noticed that the iOS project opened in the old Xcode version **(11.2.1)**, and when I build and run it again on the iOS simulator, it crashed. On the other hand, if I opened the latest installed version of Xcode **(11.4)**, and I open the Ionic - ios platform on that version, it works!

It looks like an Ionic/Capacitor needs to be opened directly on the last xCode version installed on the device.

**Reply by jcesarmobile** [reply]
No, Capacitor works on any Xcode 11+ version, I've tested on 11.0 and 11.7 without problems.
Your storyboard probably got corrupted or you had a bad Xcode install.

**Reply by aakatheeri** [reply]
Thanks for the clarifications! It works well now.
